### PR TITLE
PMM-7405 Fix tests.

### PR DESCRIPTION
**Pull request by askomorokhov**
https://jira.percona.com/browse/PMM-7405

FB: https://github.com/Percona-Lab/pmm-submodules/pull/1518

### CODE REVIEW

**Comment by BupycHuk** [comment]
@askomorokhov could you add link to FB, please?

**Comment by askomorokhov** [comment]
> @askomorokhov could you add link to FB, please?

@BupycHuk, yes, sure. Added to the description.

File: inventory/services_test.go [hunk 1]
```diff
@@ -1210,7 +1210,7 @@ func TestExternalService(t *testing.T) {
 		})
 		assert.NoError(t, err)
 		assert.NotNil(t, noFilterServicesList)
-		assert.Len(t, noFilterServicesList.Payload.Mysql, 1)
+		assert.Len(t, noFilterServicesList.Payload.Mysql, 0)
```
Comment (JiriCtvrtka) [inline inventory/services_test.go:1213]: @askomorokhov This test was broken before? Or please what changed here? Other changes look good.

Comment (askomorokhov) [reply inventory/services_test.go:1213]: Looks like it was broken before. I saw in some PR it was changed - but did not pay attention enough to it.

**Review by JiriCtvrtka — APPROVED** [review]

**Review by percona-csalguero — APPROVED** [review]

**Review by artemgavrilov — APPROVED** [review]

**Review by atymchuk — APPROVED** [review]

**Pull request merged**